app/src/main/kotlin/com/wire/android/util/ImageUtil.kt

```diff
@@ -0,0 +1,76 @@
+package com.wire.android.util
+
+import android.content.Context
+import android.graphics.Bitmap
+import android.graphics.BitmapFactory
+import android.graphics.Matrix
+import android.net.Uri
+import androidx.exifinterface.media.ExifInterface
+import java.io.ByteArrayOutputStream
+
+/**
+ * Rotates the image to a [ExifInterface.ORIENTATION_NORMAL] in case it's rotated with a different orientation
+ *
+ * @param exif Exif interface for of the image to rotate
+ * @return Bitmap the rotated bitmap or the same in case there is no rotation performed
+ */
+private fun Bitmap.rotateImageToNormalOrientation(exif: ExifInterface?): Bitmap {
+    val orientation = exif?.getAttributeInt(ExifInterface.TAG_ORIENTATION, ExifInterface.ORIENTATION_NORMAL)
+    val matrix = Matrix()
+    when (orientation) {
+        ExifInterface.ORIENTATION_NORMAL -> return this
+        ExifInterface.ORIENTATION_ROTATE_180 -> matrix.postRotate(180f)
+        ExifInterface.ORIENTATION_ROTATE_90 -> matrix.postRotate(90f)
+        ExifInterface.ORIENTATION_ROTATE_270 -> matrix.postRotate(270f)
+        else -> return this
+    }
+
+    return try {
+        val rotated = Bitmap.createBitmap(this, 0, 0, this.width, this.height, matrix, true)
+        this.recycle()
+        rotated
+    } catch (exception: Exception) {
+        this
+    }
+}
```

Comment:
kind of wonder what Normal orientation is here, without experience with ExiInterface :P maybe we could include the meaning of "normal orientation" info inside the method name ?

Reply:
Yeah, I had to research myself a couple of things regarding this `ExifInterface` (Image metadata) 🔬  
The normal orientation means the way we took the picture, for example, so if we took a picture in portrait, should not be rotated by the camera software. This will get that info accordingly to the metadata.

I added a Kdoc to the function, but didn't manage to change the naming since it will depend, Normal means either Portrait or Landscape.